examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -280,102 +281,99 @@ of detachFrom() just deletes the one and only reference to the microservice in O
-                                        + " retweeted from @user"
-                                        + Integer.toString(id)
-                                        + " in: "
-                                        + ((end - start) / 1000000)
-                                        + "ms");
-                    }
-                    continue;
                 }
+                continue;
+            }
 
-                if (event.equals("FAVORITE") && userId > 0) {
-                    long start = System.nanoTime();
-                    /* Favorite one of the last 10 tweets of some user */
-                    int id = getId(userId);
-                    List<Tweet> lastTweets = timelines.get(id).getTweets(0, 10);
-                    if (lastTweets.size() > 0) {
-                        timelines
-                                .get(id)
-                                .favorite(
-                                        lastTweets.get(gen.nextInt(lastTweets.size())).getId(),
-                                        "user" + Integer.toString(userId));
-                        long end = System.nanoTime();
-                        System.out.println(
-                                "@user"
-                                        + Integer.toString(userId)
-                                        + " favorited from @user"
-                                        + Integer.toString(id)
-                                        + " in: "
-                                        + ((end - start) / 1000000)
-                                        + "ms");
-                    }
+            if (event.equals("FAVORITE") && userId > 0) {
+                long start = System.nanoTime();
+                /* Favorite one of the last 10 tweets of some user */
+                int id = getId(userId);
+                List<Tweet> lastTweets = timelines.get(id).getTweets(0, 10);
+                if (lastTweets.size() > 0) {
+                    timelines
+                            .get(id)
+                            .favorite(
+                                    lastTweets.get(gen.nextInt(lastTweets.size())).getId(),
+                                    "user" + Integer.toString(userId));
+                    long end = System.nanoTime();
+                    System.out.println(
+                            "@user"
+                                    + Integer.toString(userId)
+                                    + " favorited from @user"
+                                    + Integer.toString(id)
+                                    + " in: "
+                                    + ((end - start) / 1000000)
+                                    + "ms");
                 }
             }
-
-            System.out.println("Done populating.");
-        } catch (Exception e) {
-            e.printStackTrace();
         }
+
+        System.out.println("Done populating.");
     }
 
     public static void main(String[] args) {
```

Comment:
throw exception from this main too when runFullDemo throws exception ?

Reply:
Agreed.

Reply:
Done.